lib/puppet/defaults.rb

```diff
@@ -126,6 +126,19 @@ def self.default_diffargs
         munge(value)
         value.to_sym
       end
+    },
+  :disable_i18n => {
+      :default => false,
+      :type    => :boolean,
+      :desc    => "If true, turns off all translations of Puppet and module
+        log messages, which affects error, warning, and info log messages,
+        as well as any translations in the report and CLI.",
+      :hook    => proc do |value|
+        if value
+          require 'puppet/gettext/stubs'
```

Comment:
Could we make this part of `disable_gettext`?

Reply:
Probably. Originally we wanted to leave that up to the caller, in part to make it more transparent. But I don't think there's a reason we _can't_ move it.

Reply:
That seems ok for parity with the puppet.rb code.